nemo/collections/asr/data/audio_to_text_lhotse_prompted.py

```diff
@@ -105,7 +106,9 @@ def get_prompt_format_fn(name: str) -> Callable[[CutSet, TokenizerWrapper, bool]
 
```

Comment:
Shall we consider moving to `PROMPT_FORMAT_FNS`, `registered_prompt_format_fn` and `get_prompt_format_fn` to ` nemo.collections.common.prompts`? Seems that these are more general than the audio to text datasets. Also, should it be `registered_prompt_format_fn` or `register_prompt_format_fn`?

Reply:
TBH I think maybe we can get rid of those entirely with just a bit more refactoring but I am reluctant to do it in this PR as the scope is already wide.